src/components/Player.js

```diff
@@ -165,6 +166,12 @@ export default class Player extends Component {
     });
   }
 
+  setPosition(style, name, value) {
```

Comment:
This function is kind of like a duplication of function `setWidthOrHeight` which just assign an value to a specific property of `style`, personally, instead of creating an new function `setPosition`, we can just rename `setWidthOrHeight` to `setStyle` and do some refactoring on it.